app/src/main/java/com/example/ridemetricsdksample/MainActivity.java

```diff
@@ -32,11 +99,21 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public void onTripStarted() {
                 // TODO set text "driving"
+                stringFile.write("TRIP STARTED \n");
```

Comment:
I suggest to write the data only to the log and remove the StringFile class because it is a sample. It will be enough.